app/src/views/private/components/drawer-item.vue

```diff
@@ -355,7 +355,7 @@ function useRelation() {
 	});
 
 	const relatedCollection = computed<string | null>(() => {
-		if (!props.junctionField) return null;
+		if (!props.junctionField) return props.collection;
```

Comment:
This change feels a tad odd. If there's no relational junction, there isn't a related collection right?

Reply:
Hmm, right! I interpreted it as "related" to the collection from which the drawer has been opened 😅 

Instead of changing this line, we should probably just adjust the logic in `isDirectusFiles` to check against `collection` from props if it's not a related collection (no junction field or related primary key).

https://github.com/directus/directus/blob/ea6b64e33e988a87f029fd7d34deb52656c80daf/app/src/views/private/components/drawer-item.vue#L239-L241

Reply:
That sounds like a better solution to me 👍🏻